docs/CONTRIBUTING.md

```diff
@@ -23,6 +23,20 @@ Once you have cloned the JsDbg repository, please follow the steps below to buil
       * Common build errors and fixes:
          * Missing Windows 10 SDK error - install the correct SDK version (should be specified in the error message) and rebuild the solution.
 
+### Building JsDbg (Linux)
+
+1. Get the [x64 .NET Core SDK binary](https://dotnet.microsoft.com/download) (click the x64 link in the left column in the table at the bottom). Instructions below and `create_package.sh` assume you extract it into `~/dotnet`.
+2. To build the server, run `~/dotnet/dotnet publish -c Release -r linux-x64` from the `server/JsDbg.Gdb` directory.
+3. The command will tell you where the output is; it should be in `bin/Release/netcoreapp2.1/linux-x64/publish` under the JsDbg.Gdb directory
+4. Add the `JsDbg.py` script to your `~/.gdbinit`. If you cloned into `~/JsDbg` it should find the binary and extensions directory automatically; otherwise you will have to edit the script and update the path. Code to add to `.gdbinit`:
+```
```

Comment:
This part isn't clear. Do we need to add both the code here and "JsDbg.py" to .gdbinit?

Reply:
Good point, I've rewritten this.